airflow/providers/apache/hive/hooks/hive.py

```diff
@@ -429,7 +429,7 @@ def load_file(
         if create or recreate:
             if field_dict is None:
                 raise ValueError("Must provide a field dict when creating a table")
-            fields = ",\n    ".join([f"`{k.strip('`')}` {v}" for k, v in field_dict.items()])
+            fields = ",\n    ".join(['`{k}` {v}'.format(k=k.strip('`'), v=v) for k, v in field_dict.items()])
```

Comment:
```suggestion
            fields = ",\n    ".join([f"`{k.strip('`')}` {v}" for k, v in field_dict.items()])
```

Should be able to do this